### PR TITLE
Post about kubelet API

### PR DESCRIPTION
**Pull request by emrahkaya**
Documents the undocumented kubelet API and methods to access this API.

### CODE REVIEW

File: _posts/2020-01-13-kubelet-api.md [hunk 1]
```diff
@@ -0,0 +1,549 @@
+---
+layout: post
+title: Kubelet API 
+author: emrahkaya
+categories: [kubernetes]
+tags: [kubernetes, kubelet, metrics, monitoring]
+---
+
+In this post, we'll describe how a pod or user can access the kubelet API available on each node of a kubernetes cluster to get information about pods (and more) on that node. We first discuss which ports are  available for this purpose, then list the available endpoints (resources) of the kubelet API. Lastly we discuss how to query the secure-port of and which authentication & authorization mechanisms are used.
```
Comment (yufufi) [inline _posts/2020-01-13-kubelet-api.md]: or **a** user

File: _posts/2020-01-13-kubelet-api.md [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+
+
+* TOC
+{:toc}
+
+# Ports
+
+kubelet gets a [long list of parameters](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options) at startup, most of which are deprecated and will be [set by using a kubelet configuration file](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/) in upcoming releases. In either way, there are two parameters that relates to ports:
```
Comment (yufufi) [inline _posts/2020-01-13-kubelet-api.md]: Just "Either way,"

File: _data/authors.yml [hunk 3]
```diff
@@ -15,3 +15,8 @@ ahmddp:
   twitter: AhmedDemirpolat
   github: ahmddp
   about: Senior Software Developer
+
+emrahkaya:
+  display_name: Emrah Kaya
+  github: emrahkaya
+  about: Sense a soul in search of answers
```
Comment (yufufi) [inline _data/authors.yml:22]: http://orcz.com/Diablo_3:_I_Sense_a_Soul_in_Search_of_Answers_Achievement ?? :)

**Review by yufufi — APPROVED** [review]

File: _data/authors.yml (hunk 3 above)
Comment (emrahkaya) [reply _data/authors.yml:22]: Yeah, sure :) . But in fact, I'm more a fan of Starcraft. https://vignette.wikia.nocookie.net/starcraft/images/9/9c/ProtossArbiterPissed00SC1.ogg/revision/latest?cb=20170508192522

**Pull request merged**